internal/tool/ssh/deploy.go

```diff
@@ -131,30 +133,38 @@ func deploy(ctx context.Context, args []string) error {
 	}
 }
 
-// copyBinaries copies the tool and the application binary to the given set
-// of locations.
-func copyBinaries(locs []string, dep *protos.Deployment) error {
+// copyBinaries copies the tool and the application binary
+// to the given set of locations. It produces a map which
```

Comment:
Returns the paths to the directories where the binaries were copied, keyed by locations.